vault/resource_database_secret_backend_connection_test.go

```diff
@@ -588,9 +586,6 @@ func TestAccDatabaseSecretBackendConnectionTemplatedUpdateExcludePassword_mysql(
 					resource.TestCheckResourceAttr(testDefaultDatabaseSecretBackendResource, "verify_connection", "true"),
 					resource.TestCheckResourceAttr(testDefaultDatabaseSecretBackendResource, "mysql.0.connection_url", testConnURL),
 					resource.TestCheckResourceAttr(testDefaultDatabaseSecretBackendResource, "mysql.0.max_connection_lifetime", "10"),
-					resource.TestCheckResourceAttr(testDefaultDatabaseSecretBackendResource, "data.%", "2"),
-					resource.TestCheckResourceAttr(testDefaultDatabaseSecretBackendResource, "data.username", testUsername),
-					resource.TestCheckResourceAttr(testDefaultDatabaseSecretBackendResource, "data.password", testPassword),
```

Comment:
So these are unrelated to this PR?

Reply:
These are to be removed since username and password are now being set on the resource config itself, though how these data fields are passed to Vault, I am slightly unclear on